libnitrokey/version.h.in

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2018 Nitrokey UG
+ *
+ * This file is part of libnitrokey.
+ *
+ * libnitrokey is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * any later version.
+ *
+ * libnitrokey is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with libnitrokey. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+ */
+
+#ifndef LIBNITROKEY_VERSION_H
+#define LIBNITROKEY_VERSION_H
+
+namespace nitrokey {
+unsigned int get_major_library_version() {
+    return @PROJECT_VERSION_MAJOR@;
```

Comment:
Implementation in header file makes a small problem - it will probably not be possible (until some compiler/optimization trick is used) to use these functions in other compilation units (e.g. in C++ API - `NitrokeyManager`) due to name's clash. Otherwise all good!
I will split it to .h and .cc and merge.

Reply:
Great! Can you have another look at the test case? I think it will fail if you are on a release commit, see [this comment](https://github.com/Nitrokey/libnitrokey/commit/57c0b8c1235c5bff3640e85a8e17be20e8f422f3#diff-754854e7c0568d4b62eb6f579b2f0f50R172).